django-stubs/db/models/expressions.pyi

```diff
@@ -199,8 +202,15 @@ class ExpressionList(Func):
 
 class OrderByList(Func): ...
 
-class ExpressionWrapper(Expression):
-    def __init__(self, expression: Q | Combinable, output_field: Field) -> None: ...
+_E = TypeVar("_E", bound=Q | Combinable)
+
+class ExpressionWrapper(Expression, Generic[_E]):
+    def __init__(self, expression: _E, output_field: Field) -> None: ...
+    expression: _E
+
+class NegatedExpression(ExpressionWrapper, Generic[_E]):
```

Comment:
If you do it like this, then the _E argument of ExpressionWrapper and _E argument of NegatedExpression won't be connected.

For example since `.expression` member comes from ExpressionWrapper
```python
from django.db.models.expressions import NegatedExpression, Exists
x: NegatedExpression[Exists]
reveal_type(x.expression)  # note: Revealed type is "Any"
```

This fixes it:
```suggestion
class NegatedExpression(ExpressionWrapper[_E]):
```

Reply:
This kind of mistake would be harder to make if we used `disallow_any_generics` (#1339)